axiom/tests/test_log_events.c

```diff
@@ -19,120 +20,138 @@
 #define LOG_MESSAGE_1 LOG_MESSAGE " 1"
 #define LOG_MESSAGE_2 LOG_MESSAGE " 2"
 #define LOG_MESSAGE_3 LOG_MESSAGE " 3"
+#define LOG_MESSAGE_4 LOG_MESSAGE " 4"
 #define LOG_TIMESTAMP 12345
 
 static nr_log_event_t* create_sample_event(const char* message) {
   nr_log_event_t* e = nr_log_event_create();
   nr_log_event_set_log_level(e, LOG_LEVEL);
   nr_log_event_set_message(e, message);
   nr_log_event_set_timestamp(e, LOG_TIMESTAMP * NR_TIME_DIVISOR_MS);
+  nr_log_event_set_priority(e, 0);
   return e;
 }
 
 static void test_events_success(void) {
   /* Normal operation */
-  nr_analytics_events_t* events = NULL;
+  nr_log_events_t* events = NULL;
   nr_log_event_t* e = NULL;
-  nr_random_t* rnd = nr_random_create_from_seed(12345);
-  const char* json;
+  void* test_e;
+  nr_vector_t* vector = NULL;
+  bool pass;
+  char* json;
   const char* expected;
 
   events = nr_log_events_create(10);
   tlib_fail_if_null("events created", events);
 
   e = create_sample_event(LOG_MESSAGE_0);
```

Comment:
Would it make sense to extend `create_sample_event` to become `create_sample_event_with_priority` that would take priority as an additional argument? This way the code would read like this:
```
e = create_sample_event_with_priority(LOG_MESSAGE_0, 3);
nr_log_events_add_event(events, e);
```
and almost match the explaining comment below.

Reply:
I'd rather leave this as is - I think the code legibility is fine.